pkg/logs/client/destination.go

```diff
@@ -74,3 +85,35 @@ func (d *Destination) Send(payload []byte) error {
 
 	return nil
 }
+
+// SendAsync sends a message to the destination without blocking. If the queue is full, the incoming messages will be
+// dropped
+func (d *Destination) SendAsync(payload []byte) {
+	d.once.Do(func() {
```

Comment:
Do we really need lazy loading here ?

Reply:
We don't _really_ need this, but this avoid having only a blocked goroutine in most of our client cases (that don't use additional endpoints)